OasisDEX/oasis-borrow
# TRM labs integration

# [TRM labs integration](https://app.shortcut.com/oazo-apps/story/3817/integrate-trm-labs-api)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added risk verification after ToS flow
  
## How to test 🧪
  <Please explain how to test your changes>

- use different kind of wallets and addresses and check whether risk is verified within network tab
- check in db whether additional information are stored in tos_approval (chain_id, signature, message)
- on goerli wallet_risk table in db shouldn't be updated as for test network we always return `isRisky: false`
- on mainnet wallet_risk table should be updated
    
## Definition of done ✔️
